wiki/Tournaments/de.md

```diff
@@ -1,3 +1,8 @@
+---
+outdated_translation: true
+outdated_since: b75cab0564e5b20a69771d98bbd25edc426b3dad
```

Comment:
Tournament index translations don't need to be outdated as you can copy the table heading and date formatting of other tournaments.